public/audits/index.html

```diff
@@ -624,7 +624,6 @@ <h5 class="mb-0 mb-1 text-danger">ABDK Consulting</h5>
 ></script>
 
 <script async src="/assets/js/theme.js"></script>
-<script src="/assets/js/add-trailing-slash.js"></script>
```

Comment:
Is it ok that we remove it in many files?

Reply:
I see that we remove file so I assume this is ok

Reply:
Yes, this script is not necessary.